dashboard/app/views/layouts/_survey.html.haml

```diff
@@ -1,41 +1,55 @@
 -# English only for now
 - if language == "en"
```

Comment:
Ah... I see it's a partial in layouts -- does that mean it can appear on any page? If so you could put the method in the ApplicationHelper